{{cookiecutter.project_slug}}/pytask.ini

```diff
@@ -1 +1,2 @@
 [pytask]
+infer_latex_dependencies = false
```

Comment:
@tobiasraabe, setting this to `true` leads to:

```
===============================================================================
---------------------------------------------------------------------------------------------------------------------------------------------------- Captured stderr call -----------------------------------------------------------------------------------------------------------------------------------------------------
_pytask.exceptions.ResolvingDependenciesError: There are some dependencies missing which do not exist and are not produced by any task. See the following dictionary with missing nodes as keys and dependent tasks as values.

{'research_project/bld/figures/schelling_baseline.eps': ['research_project/src/paper/task_paper.py::task_compile_documents[depends_on0-produces0]'],
 'research_project/bld/figures/schelling_baseline.jpeg': ['research_project/src/paper/task_paper.py::task_compile_documents[depends_on0-produces0]'],
 'research_project/bld/figures/schelling_baseline.pdf': ['research_project/src/paper/task_paper.py::task_compile_documents[depends_on0-produces0]'],
 'research_project/bld/figures/schelling_baseline.ps': ['research_project/src/paper/task_paper.py::task_compile_documents[depends_on0-produces0]'],
 'research_project/bld/figures/schelling_max_moves_2.eps': ['research_project/src/paper/task_paper.py::task_compile_documents[depends_on0-produces0]'],
 'research_project/bld/figures/schelling_max_moves_2.jpeg': ['research_project/src/paper/task_paper.py::task_compile_documents[depends_on0-produces0]'],
 'research_project/bld/figures/schelling_max_moves_2.pdf': ['research_project/src/paper/task_paper.py::task_compile_documents[depends_on0-produces0]'],
 'research_project/bld/figures/schelling_max_moves_2.ps': ['research_project/src/paper/task_paper.py::task_compile_documents[depends_on0-produces0]']}
=================================================================================================================================================== short test summary info ===================================================================================================================================================
FAILED tests/test_run_project.py::test_pytask_python - AssertionError: assert 4 == 0
```

Seems like the scanner wants to see **all** possible graphics files where one of them would be enough?

Reply:
Does this PR resolve the issue? https://github.com/pytask-dev/pytask-latex/pull/16

Reply:
Thanks for the quick fix!!!

Seems like I am unable to get `pytask` to use the development version of `pytask-latex`. Whatever variant of `conda develop` I tried, it always shows `Plugins: latex-0.0.9` and the familiar error.

You could try yourself by running `pytest tests/test_run_project.py` on this PR, though I gather from [here](https://github.com/pytask-dev/pytask-latex/pull/16/commits/6ad9df3fd50f7ac79f269216ac19ff4fdf6a135c) that the whole scanner may not work on Windows?

Reply:
I have not updated the version number yet. So 0.0.9 should be fine, but it should say develop instead of the pytask channel.

Reply:
Regarding the second point: I use tinytex which does not deploy to Windows and so all tests are skipped. But latexmk works on windows.

Reply:
(thought I had sent the last message off yesterday... no rush)

Reply:
Make an editable install from the PR branch. Maybe you need to do it with pip install -e instead of conda develop. conda list pytask should then report <develop>.

Reply:
Excellent -- test passes!